packages/foam-vscode/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "lib": ["ES2019", "es2020.string", "DOM"],
     "sourceMap": true,
     "strict": false,
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "skipLibCheck": true
```

Comment:
what's the goal of this setting?

Reply:
Preventing `tsc` to check the types within the project. I've added this to prevent the collisions between `mocha` and `jest`. We use `mocha` for the e2e tests and `jest` for the vscode extension. They are incompatible with each other, but for our use case that's not a problem. However, `tsc` scans the entire `node_modules` folder and detects the collisions.